resources/assets/lib/stores/user-store.ts

```diff
@@ -27,25 +27,16 @@ export default class UserStore implements DispatchListener {
   }
 
   @action
-  getOrCreate(userId: number, props?: UserJson): User {
+  getOrCreate(json: UserJson): User {
```

Comment:
this looks more like `updateOrCreate` 🤔 (or maybe just `update` with implied create if missing)